src/app/activities/[activityId]/sections/SubmitSection.js

```diff
@@ -0,0 +1,76 @@
+"use client";
+
+import React from "react";
+import {Button, Typography, Box, Checkbox, Container} from "@mui/material";
+import Link from "@/app/(homepage)/components/Link";
+
+function FormBlock() {
+
+  return (
+    <Box>
+      <Typography
+        variant="h3"
+        gutterBottom
+        sx={{
+          textAlign: {xs: 'center', lg: 'left'},
+          width: '100%'
+        }}
+      >
+        Let&apos;s get your activities live and in front of parents!
+      </Typography>
+      <Box sx={{
+        display: 'grid',
```

Comment:
Grid is an overkill here, we have a one-dimensional layout, flex will be enough

Reply:
I remember, but with flex we can't centre buttons. justifyContent centre work only with grid

Reply:
justifyContent works for main axis. In case of flexDirection column justifyContent means vertical alignment. "center" button means horizontal align. Horizontal align is secondary axis for column flexDirection. For secondary axis alignItems CSS property is used.